spec/defines/job_spec.rb

```diff
@@ -135,6 +135,21 @@
     end
   end
 
+  context 'job with sensitive environment' do
+    let(:params) do
+      {
+        command: '/bin/true',
+        environment: [sensitive('KEY=value')]
+      }
+      let(:cron_timestamp) { get_timestamp(params) }
```

Comment:
the let needs to be outside the let params block.

Reply:
well, a simple `bundle update` allowed me to run the tests on my side